src/brsTypes/BrsType.ts

```diff
@@ -398,6 +398,6 @@ export class Uninitialized implements BrsValue, Comparable {
     }
 
     toString(parent?: BrsType) {
-        return "<UNINITIALIZED>";
+        return "<uninitialized>";
```

Comment:
Printing an uninitialized variable does print the capitalized version in the reference BrightScript implementation from what I can tell — would you mind reverting this portion?

```suggestion
        return "<UNINITIALIZED>";
```